public/test/core/redux/reducerTester.ts

```diff
@@ -66,9 +66,9 @@ export const reducerTester = <State>(): Given<State> => {
     disableDeepFreeze = false
   ): When<State> => {
     reducerUnderTest = reducer;
-    initialState = { ...state };
+    initialState = state;
     if (!disableDeepFreeze) {
-      initialState = deepFreeze(initialState);
+      deepFreeze(initialState);
```

Comment:
Great catch ⭐

Reply:
I think we need to be a bit more clever here too. Only deepFreeze if this is an object.

Reply:
i think it make sense to freeze array as well as this is valid:
```ts
const obj = [1, 2, 3];
obj.somethingElse = 4;
```

what do you think?

Reply:
Arrays are object so yes ;)